### DISCUSSION TITLE
Design token distribution per platform

### DISCUSSION THREAD

**Discussion by ptu14** [six7/figma-tokens#1856] [Q&A]
Question for development teams and/or tokens studio team.

How do you integrate tokens in your applications?

We have a project for 3 different platforms:
- android (need xml)
- iOS (need swift class)
- web (need css)

Tokens in `json` are synchronized with one repository. For now, we use `style-dictionary` in this repo, to automatically generate all platforms files, and then pull them from this repo into specific project. 

How do you deal with generating tokens per platform? What is the best practice? Maybe generators should be in individual per projects, and we should only pull tokens in `json`?

**Comment by six7** [comment]
We use style-dictionary to generate for individual platforms with the help of https://www.npmjs.com/package/@tokens-studio/sd-transforms - we're looking for feedback on that module on what you'd need for multiplatform defaults 🙏
